docs/contract-interface/payable-methods.md

```diff
@@ -41,8 +41,8 @@ impl Contract {
         env::log_str("Thanks!");
     }
     pub fn do_not_take_my_money(&mut self) {
-        if near_sdk::env::attached_deposit() != 0 {
-            near_sdk::env::panic(b"Method do_not_take_my_money doesn't accept deposit");
+        if env::attached_deposit() != 0 {
+            env::panic_str("Method do_not_take_my_money doesn't accept deposit");
```

Comment:
the prefix of `near_sdk` was intentional. That is what the code gets expanded to. Might be a bit misleading if someone thinks they need to import `near_sdk::env` for this to work

Reply:
Fixed